tests/testthat/test-remove_stop_words.R

```diff
@@ -0,0 +1,28 @@
+test_that("remove_stop_words() removes stop words", {
```

Comment:
In the Milestone instructions, it says that we should group test cases in a way that is sensible so that what is being tested matches the string associated with the test_that code blocks. Is it possible to adjust your tests to group them somehow? Otherwise looks great!